wrecker.go

```diff
@@ -61,21 +62,33 @@ func (w *Wrecker) Delete(endpoint string) *Request {
 }
 
 func (w *Wrecker) sendRequest(r *Request) (*http.Response, error) {
-	var contentType string
+
+	var contentType string = "application/x-www-form-urlencoded"
```

Comment:
Are these values enumerated anywhere in the Go packages?  If they are we should use them.  Definitely not something that's blocking the merge of this PR though